rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -1071,6 +1072,14 @@ data class DataStructValue(var value: String, private val optionalExternalLen: I
 fun Int.asValue() = IntValue(this.toLong())
 fun Boolean.asValue() = BooleanValue(this)
 
+infix fun Value.distanceFrom(other: Value): Long {
+    val a = this.asInt()
+    val b = other.asInt()
+    return (b - a).value
+}
+
+infix fun Value.absoluteDistanceFrom(other: Value): Long = abs(this distanceFrom other)
```

Comment:
I think it is never used, just like `distanceFrom`. Do you believe it could be useful in the future?

Reply:
You are right. I think it is a leftover of one of the first draft implementation of the binary search strategy but as of now they are unused. I removed them in the latest commit as they are so easy to implement that I think rewriting them if needed in future outweights keeping them unreferenced in the code for no concrete present reason.